kore/src/Kore/Repl/Interpreter.hs

```diff
@@ -463,6 +463,40 @@ tryAxiomClaim eac = do
     singleton :: a -> [a]
     singleton a = [a]
 
+    showUnificationFailure
+        :: Either [Axiom level] [Claim level]
+        -> Graph.Node
+        -> ReplM level ()
+    showUnificationFailure ac node = do
+        let rw =
+                listToMaybe
```

Comment:
I would suggest extracting all or at least part of this:
```.hs
listToMaybe . fmap (left . getRewriteRule) . either (fmap unAxiom) (fmap unClaim)
```
I can't really tell what's going on because this is at a very different level of detail than the rest of the function.